packages/charrua-core/charrua-core.0.5/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/mirage/charrua-core/releases/download/v0.5/charrua-core-0.5.tbz"
```

Comment:
The naming scheme changed, can we keep the same on opam for consistency ?
https://github.com/mirage/charrua-core/archive/v0.5.tar.gz